src/registry_schemas/schemas/court_order.json

```diff
@@ -0,0 +1,34 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/court_order",
+    "type": "object",
+    "title": "Court Order Information Schema",
+    "properties": {
+        "fileNumber": {
+            "type": "string",
+            "minLength": 5,
+            "maxLength": 20,
+            "description": "The court assigns each court order a unique file number up to 20 characters in length."
+        },
+        "orderDate": {
+            "type": "string",
+            "description": "The date of the court order in the ISO 8601 format YYYY-MM-DDThh:mm:ssTZD.",
```

Comment:
The format used to date by the API is not ISO 8601 (but it's close). Example:
"1970-01-01T00:00:00+00:00"

I think this description be updated (and maybe add an `examples` property - see other schemas for examples.

Reply:
actually, it is, we're just a little forgiving

> ISO 8601 uses the “T” character to separate the date and time. In RFC 3339, you can replace the “T” character with only using space.

https://github.com/bcgov/business-schemas/blob/master/requirements/prod.txt#L4

Reply:
I added an example property.